spirit/category/migrations/0006_category_sort.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2018-12-26 08:48
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def reorder(apps, schema_editor):
```

Comment:
This needs tests as well. See, https://github.com/nitely/Spirit/blob/master/spirit/user/tests.py#L33 for how to test a data migration